pkg/dockerregistry/server/blobdescriptorservice.go

```diff
@@ -40,28 +40,32 @@ func (bs *blobDescriptorService) Stat(ctx context.Context, dgst digest.Digest) (
 	if err == nil {
 		return desc, nil
 	}
-
 	context.GetLogger(ctx).Debugf("(*blobDescriptorService).Stat: could not stat layer link %s in repository %s: %v", dgst.String(), bs.repo.Named().Name(), err)
 
+	// we couldn't find the layer link
```

Comment:
comment should elucidate the implications of not finding the layer link

Reply:
this is just a refactor of the method, i'm happy to make better comments